pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/samalba/dockerclient"
```

Comment:
@KennethNagin To clarify - You should avoid using dockerclient structures in YOUR code even if it is already beaning used in that file.  I understand that using engine-api/types in YOUR code may require you to change existing (not yours) code as well.  But it's not reasonable to add more code that someone else later on will need to change as well. 
It's OK that you make changes that are related only to structures your are using .